### ISSUE TITLE
Problem: DIP upload rsync command populated with "None" on deployment

### ISSUE THREAD

**Issue by sallain** [artefactual/archivematica#879]
In the AtoM DIP upload configuration screen, there's a parameter for adding an rsync command. It looks like this field is being automatically populated with the text string "None", which causes DIP upload to AtoM to fail.
I suspect that this value is present when Archivematica is deployed, but this should be checked, as well as if it reappears after an upgrade. If it does, we should fix this so that the field is left blank.

**Comment by jrwdunham** [comment]
This should be fixed by https://github.com/artefactual/archivematica/pull/896. Waiting for code review.

**Comment by sromkey** [comment]
We may need a totally fresh deploy to test this because I think jenkins is deploying our VMs with the DIP upload info pre populated with our test AtoM credentials. @mamedin could you please confirm?

**Comment by mamedin** [comment]
Yes @sromkey, a saved database is used when upgrading the AM QA sites with jenkins. These saved configurations don't have this "None" string.

I have tested a new deploy in a new OVH VM using commit `ef35d8f821593d28e4dad5e55fa3f357f041c047` (
It is just the commit prior to this PR) and I could see the "None" string in the DIP upload config. I upgraded to commit `a9e836d62e19e8d3e65a8077334e01c97b377b09` (this PR) and this "None" string disappeared.
